swap/src/cli/command.rs

```diff
@@ -324,6 +347,24 @@ pub enum RawCommand {
         #[structopt(flatten)]
         bitcoin: Bitcoin,
     },
+    ListSellers {
+        // TODO: sane default value
+        #[structopt(
+            long,
+            help = "The peer-id of a rendezvous node that sellers register with"
+        )]
+        rendezvous_node_peer_id: PeerId,
+
+        // TODO: sane default value
+        #[structopt(
+            long,
+            help = "The multiaddr of a rendezvous node that sellers register with"
+        )]
+        rendezvous_node_addr: Multiaddr,
```

Comment:
A single multi-addr should be good enough here.

For now we can assume that dialling the `/dnsaddr` will only return a single result. We will deal with that returning multiple results at another time.

Reply:
Actually, by appending `/p2p` to it, only a single address will be dialed. See what I did for the ASB.